docs/release-notes/2.31.0.md

```diff
@@ -0,0 +1,164 @@
+
+All previous Integrated Alerting and Grafana Alerting functionality are now consolidated in a single pane of glass on the **Alerting** page. From here, you can configure, create and monitor alerts based on Percona or Grafana templates. 
+
+The **Alert Rules** tab has also evolved into a more efficient interface with an added layer for simplifying complex Grafana rules. You’ll find that the new **Percona templated alert** option here offers the same functionality available in the old Integrated Alerting, but uses a friendlier interface, powered by Grafana’s advanced alerting capabilities. 
+
+As an important and generally useful feature, this new Alerting feature is now enabled by default and ready to use in production! 
+
+### Deprecated Integrated Alerting
+The new Percona Alerting feature fully replaces the old Integrated Alerting in previous PMM versions. The new alerting brings full feature parity with Integrated Alerting, along with additional benefits like Grafana-based alert rules and a unified alerting command center. 
+
+If you have been using Integrated Alerting in the previous PMM version, your custom alert rule templates will be automatically migrated to PMM 2.31. After upgrading to this new version, you will find all your alert templates under **Alerting > Alert Templates**. 
+These templates are also available for creating new alert rules using the new Percona Templated Alerts option on the Alert Rules tab. 
+
+However, custom alert rules created with Integrated Alerting are not automatically migrated to Percona Alerting. After upgrading, make sure to manually migrate any custom alert rules that you want to transfer to PMM 2.31 using the script that will be provided soon. 
+
+
+## Vacuum monitoring dashboard
+
+!!! caution alert alert-warning "Important"
+    This experimental dashboard is subject to change. It is recommended to use this dashboard for testing purposes only.
+
+
+A new experimental dashboard was released to help our users gain timely insights into the autovacuum process in PostgreSQL. This dashboard is designed to help fine-tune the vacuum configuration and to prevent a XID wraparound - both of which directly contribute to improved performance of the database.
+
+This dashboard contains the following:
+
+- Dead tuples - Identifies the number of dead rows in each table even though the rows are physically removed from the table.
+
+- Last time vacuum ran - Tracks the last time a vacuum or autovacuum process successfully ran on each of your tables.
+
+- Number of rows modified since last Analyze - The number of rows changed since the last time ANALYZE ran.
+
+- Manual vacuum events - Tracks the number of times a manual vacuum was run on each table. 
+
+- Table disk usage - Tracking the disk space used by each table is crucial as it enables you to gauge expected changes in the query performance over time—but it can also help you detect potential vacuuming-related issues.
```

Comment:
```suggestion
- Table disk usage - Tracking the disk space used by each table is crucial as it enables you to gauge expected changes in the query performance over time - but it can also help you detect potential vacuuming-related issues.
```